.github/workflows/test.yml

```diff
@@ -7,9 +7,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with:
-        node-version: '14'
-    - name: NPM version
-      run: npm --version
+        node-version: '18'
```

Comment:
It's true that this value should be updated, but can you keep your PR narrowly-scoped to just the very thing you're trying to update?